frontend/src/components/SearchByCriteria.js

```diff
@@ -1,5 +1,121 @@
+            difficulty: '',
+            categories: [],
+            picture: false,
+          }}
+        >
+          {({ values }) => (
+            <Form>
+              <div>
+                <h3 className="myFormTitle">A recept neve </h3>
+                <Field
+                  className="col form-control-lg"
+                  type="text"
+                  name="name"
+                  placeholder="Csilis bab"
+                />
+              </div>
+              <div className="mt-4 col">
+                <h3 className="myFormTitle">Nehézség </h3>
+                <Field
+                  className="col custom-select-lg"
+                  as="select"
+                  required
+                  name="difficulty"
+                >
+                  {levels
+                    ? levels.map((level) => (
+                        <option key={level} value={level}>
+                          {level}
+                        </option>
+                      ))
+                    : 'loading'}
+                </Field>
+              </div>
+              <div className="form-control mt-5" id="my-radio-group">
+                <h3 className="myFormTitle">Elkészítés idő </h3>
+                <div
+                  className="row"
+                  role="group"
+                  aria-labelledby="my-radio-group"
+                >
+                  {times.map((time) => (
+                    <div className="col-4" key={time}>
+                      <Field type="radio" name="preparationTime" value={time} />
+                      {time - 30} - {time - 1} perc
+                    </div>
+                  ))}
+                </div>
+              </div>
+              <div>
+                <div role="group" aria-labelledby="checkbox-group">
+                  <h3 className="myFormTitle">Kategóriák </h3>
+                  <div
+                    className="row"
+                    role="group"
+                    aria-labelledby="categories"
+                  >
+                    {categoryList.map((l) => (
+                      <div className="col-2" htmlFor={l.name}>
+                        {l.name}
+                        <div>
+                          <Field
+                            type="checkbox"
+                            name="categories"
+                            value={l.name}
+                          />
+                        </div>
+                      </div>
+                    ))}
+                  </div>
+                </div>
+              </div>
+              <div className="">
+                <h5 className="mt-5">
+                  Legyen kép a receptről?
+                  <Field className="ml-5" type="checkbox" name="picture" />
+                </h5>
+              </div>
+              <Link
+                className="btn btn-success mt-3"
+                type="submit"
+                to={{
+                  pathname: '/searchResultByCriteria',
```

Comment:
Szerintem egyszerűbb, ha nem nyílik külön oldal, hanem a form alatt lenne az eredmények listája.

Reply:
Megpróbálhatjuk, azért így lett mert így volt a feladat kiírásban hogy egy új eredmányek oldalra menjünk

Reply:
Igen, úgy lett elkészítve, ahogy az issueban kértem.  
Viszont nem tartom rossz ötletnek azt, hogy a maradjunk ezen az oldalon, így a kritériumok változtatásának lehetőségét se kell újra belehúzni, szval engem megvettél az ötlettel Roland. Annyi lenne ebben az esetben a kérés, hogy a már beadott kritériumokat tartsuk meg a formban is, hogy látszódjon, hogy mi a jelenlegi keresés.
A kérdésem annyi, hogy ez a ticeket most menjen-e be így és a listás issue-ba rakjuk bele, hogy maradjon az oldalon, vagy már itt intézzük el?

Reply:
Most azt beszéltük , hogy #44 ben már megírták a megjelenést , van külön komponens , és majd ők megcsinálják azzal a komponenssel hogy ugyan úgy nézzen ki

Reply:
Az 'ők' kik, és melyik ticketen? :) (Tippre Jonkáék és abban a ticketben?)
És akkor szeretnéd, ha ez most így menne és az pedig egy refaktort is tartalmazna erre a részre? Jól értem?

Reply:
Igazából, mivel van ennek az oldalnak is külön ticketje, lehet arra adminisztrálni (https://github.com/flowacademyhu/kappa-konyhatunder/issues/8). Legyen ott a megvalósítása ennek az oldalnak, ahhoz kapcsolódó pr legyen. Tudom, h a másikban van a komponens, de annak a beküldése után itt is használható lesz. Mire ezen az oldalon eljuttok a lista behúzásához, addigra hopefully az már bent lesz (vagy más megoldással húzzátok be). Ennél a listánál lehet azért majd a https://github.com/flowacademyhu/kappa-konyhatunder/issues/8 issuen dolgozni FE oldalról, hogy ne legyen törölve/legyen beállítva a keresési kritérium, ha volt